build.gradle

```diff
@@ -79,7 +79,6 @@ subprojects {
         forkEvery = 50
         maxParallelForks = 1
         minHeapSize = "128m"
-        maxHeapSize = "512m"
```

Comment:
Funny cause this option is set to 8G just 4 lines above, so I wondered which one prevailed.